src/main/java/reposense/model/FileTypeManager.java

```diff
@@ -19,7 +19,10 @@ public FileTypeManager(List<FileType> formats) {
         groups = new ArrayList<>();
     }
 
-    public String getFileType(String fileName) {
+    /**
+     * Returns the assigned file type {@code label} of the file {@code fileName}.
+     */
+    public String getFileTypeLabel(String fileName) {
```

Comment:
Actually on a second thought, why don't you just return `FileType` directly? When used on a `String` parameter input, it will automatically calls `.toString()` to convert it to a `String` type variable.

Reply:
This can be done also. Will get the method to return FileType directly